extensions/wikia/CommunityPage/styles/components/_insights-module.scss

```diff
@@ -1,77 +1,86 @@
 @import 'skins/shared/color';
-@import 'skins/oasis/css/core/breakpoints-variables';
 @import 'skins/shared/mixins/flexbox';
 
-$item-icon-size: 34px;
-$item-icon-margin-right: 6px;
-$item-data-width: calc(70% - #{$item-icon-size + $item-icon-margin-right});
-$item-link-width: 30%;
+$content-spacing: 18px;
+$content-spacing-large: 24px;
+$element-size: 48px;
+$font-size-small: 12px;
+$font-size-regular: 14px;
+$icon-size: 18px;
 
-$edit-icon: '/extensions/wikia/CommunityPage/images/edit-pen.svg';
+.community-page-insights-module {
+	@include flexbox();
+	@include flex-direction(column);
 
-@if $is-dark-wiki {
-	$edit-icon: '/extensions/wikia/CommunityPage/images/edit-pen-dark.svg';
-}
+	background-color: $color-page;
+	border: 1px solid $color-page-border-subtle;
+	margin-bottom: $content-spacing;
+	width: 239px;
 
-.community-page-insights-module {
-	margin-bottom: 34px;
+	.community-page-insights-module-article-data {
+		color: $color-links;
+	}
 
-	.community-page-insights-module-header {
-		font-size: 24px;
-		font-weight: bold;
-		line-height: 28px;
-		margin: 0 0 12px;
+	.community-page-insights-module-footer {
+		@include flexbox();
 	}
 
-	.community-page-insights-module-list {
-		list-style: none;
-		margin: 0;
+	.community-page-insights-module-full-list-link {
+		@include flex-grow(1);
+
+		color: $color-buttons;
+		display: block;
+		font-size: $font-size-small;
+		font-weight: bold;
+		line-height: $element-size;
+		margin-right: $content-spacing-large;
+		text-align: right;
+		text-transform: uppercase;
 	}
 
-	.community-page-insights-module-list-item {
+	.community-page-insights-module-header {
+		@include align-items(center);
 		@include flexbox();
 
-		align-items: center;
-		border-bottom: 1px solid $color-page-border-subtle;
-		padding: 20px;
+		background: $color-buttons;
+		color: $color-button-text;
+		font-size: $font-size-regular;
+		font-weight: bold;
+		line-height: 16px;
```

Comment:
In general this could also go into $insights-header-line-height (to keep all the measurements in one place)